.github/workflows/lint.yaml

```diff
@@ -48,7 +48,7 @@ jobs:
         env:
           ENABLE_BPMETADATA: 1
 
-      - run: docker run --rm -e ENABLE_BPMETADATA -v ${{ github.workspace }}:/workspace ${{ steps.variables.outputs.dev-tools }} /usr/local/bin/test_lint.sh
+      - run: docker run --rm -e ENABLE_BPMETADATA -e EXCLUDE_LINT_DIRS -v ${{ github.workspace }}:/workspace ${{ steps.variables.outputs.dev-tools }} /usr/local/bin/test_lint.sh
         env:
           ENABLE_BPMETADATA: 1
-
+          EXCLUDE_LINT_DIRS: \./app/java/.mvn/wrapper/maven-wrapper.jar
```

Comment:
I am not sure why the cloud-foundation-bot made this commit: https://github.com/GoogleCloudPlatform/terraform-cloud-client-api/commit/e7e96a6057f98f6f1e4ba2a3d6ae3541c6ced9a2.

So I've created https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/2338 as a long-term solution.

Reply:
GoogleCloudPlatform/cloud-foundation-toolkit#2338 is now merged.

Reply:
Today, I learned, to change the value of `EXCLUDE_LINT_DIRS`, you must update the Terraform here: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/terraform/test-org/org/locals.tf#L151

Neat.